openmc/weight_windows.py

```diff
@@ -612,3 +613,21 @@ def wwinp_to_wws(path):
         wws.append(ww)
 
     return wws
+
+
+def hdf5_to_wws(path):
+    """Create WeightWindows instances from a weight windows hdf5 file
+
+    .. versionadded:: 0.13.1
+
+    Parameters
+    ----------
+    path : str or pathlib.Path
+        Path to the weight windows hdf5 file
+
+    Returns
+    -------
+    list of openmc.WeightWindows
+    """
+    with h5py.File(path) as h5_file:
+        return [WeightWindows.from_hdf5(ww) for ww in h5_file['weight_windows']]
```

Comment:
I think this new hdf5_to_wws function is calling ```WeightWindows.from_hdf5``` with just one of the two arguments required. I think another argument is needed for the meshes